client/src/components/application/Examine/RequestInfoHeader.vue

```diff
@@ -624,6 +624,12 @@
           return this.$store.getters.requestType
         },
         set(value) {
+          const requestType = this.requestType_options.find(x => x.value === value)
```

Comment:
This too will return undefined if not found. Does it need a fallback value?

Reply:
I believe this is ok because it's the setter and we always have `value` but you are right, if there is a mapping issue.
I will look at this though to see if i can add some safety.

Reply:
Good safety check, thanks.